### PR TITLE
Update images to address inconsistencies

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

File: en/docs/services/create-your-first-service.md [hunk 1]
```diff
@@ -153,10 +153,12 @@ To deploy the `covid-stats` service, follow this procedure:
 
     ![Deployment Logs](../assets/img/services/deployment-logs.png){.cInlineImage-full}
 
-3. Once the service is deployed, click the **Go Live** tab below the low-code diagram. Then click **://cURL** and copy the cURL command.
+3. Once the service is deployed, click on **://cURL**.
 
     ![Get cURL command](../assets/img/services/copy-curl-command.png){.cInlineImage-full}
```
Comment (shaniR) [inline en/docs/services/create-your-first-service.md:158]: There is not curl command here. Is this the correct Image?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/services/create-your-first-service.md [hunk 2]
```diff
@@ -153,10 +153,12 @@ To deploy the `covid-stats` service, follow this procedure:
 
     ![Deployment Logs](../assets/img/services/deployment-logs.png){.cInlineImage-full}
```
Comment (Nashaath) [inline en/docs/services/create-your-first-service.md:154]: Need to follow sentence-style for alt text.

File: en/docs/services/create-your-first-service.md (hunk 1 above)
Comment (RukshiW) [reply en/docs/services/create-your-first-service.md:158]: Yes. The curl command shows in the next dialog box that opens. The image just shows the **://cURL** button. The text that directs the user to copy the curl command comes after the image.

File: en/docs/services/create-your-first-service.md (hunk 2 above)
Comment (RukshiW) [reply en/docs/services/create-your-first-service.md]: Corrected this.

**Comment by RukshiW** [comment]
Corrects ttps://github.com/wso2-enterprise/choreo/issues/4454

**Review by shaniR — APPROVED** [review]

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**